plugins/catalog-backend/api-report.md

```diff
@@ -528,6 +522,11 @@ export class CommonDatabase implements Database {
   ): Promise<DbEntityResponse>;
 }
 
+// @public
+export const createCatalogPermissionRule: <TParams extends unknown[]>(
```

Comment:
Just to double check, this is actually supposed to be public and would be used by anyone that wants to create their own rules right? Seems like a good pattern

Reply:
Correct - there's an example of third-party catalog rule in the mob/authorization-framework branch if you're interested.